11-parameter-store/README.md

```diff
@@ -121,6 +121,8 @@ entire subtree with a single query.
 _When you look at your stack in the CloudFormation console, can you find
 the values of your parameter resources there?_
 
+> !GetAtt Param.Value lets you do this
```

Comment:
FYI, the question was asking about the CFN console/website.  The CFN console will show you the values of stack parameters as they were resolved during the last change set. Unless they values are SSM Secure Strings.